rpc/wrtc_signaling_answerer.go

```diff
@@ -71,6 +76,11 @@ func newWebRTCSignalingAnswerer(
 		bgWorkers:    bgWorkers,
 		logger:       logger,
 	}
+	if options.externalSignalingConn != nil {
+		ans.conn = options.externalSignalingConn
+		ans.sharedConn = true
+	}
+	return ans
```

Comment:
I know we use different opts for the internal signaling answerer, but there should be a check here to check whether this new signaling answerer is internal or external.
or we could rename the option to `signalingConn` and then explicitly call this out when we create the signaling answerers